cssom-1/Overview.bs

```diff
@@ -2353,20 +2353,20 @@ To <dfn export>serialize a CSS declaration block</dfn> <var>declaration block</v
    <li><i>Shorthand loop</i>: For each <var>shorthand</var> in <var>shorthands</var>, follow these substeps:
     <ol>
      <li>Let <var>longhands</var> be an array consisting of all <a>CSS declarations</a> in <var>declaration block</var>'s
-     <a for="CSSStyleDeclaration">declarations</a> that that are not in <var>already serialized</var> and have a
+     <a for="CSSStyleDeclaration">declarations</a> that are not in <var>already serialized</var> and have a
      <a for="CSS declaration">property name</a> that
      maps to one of the shorthand properties in <var>shorthands</var>.
```

Comment:
Should this be "that maps to shorthand"? The loop is only concerned about that, not the entire list of possible shorthands.

Reply:
I've pushed a commit to correct this.

Reply:
Actually I've undone that commit. I don't understand why we have both `longhands` and `current longhands`. `longhands` only gets used to then copy into `current longhands`. Maybe it's supposed to only be created once instead of each time in `shorthand loop`?